test/doc/content/source/userobjects/InterfaceUO_QP.md

```diff
@@ -0,0 +1,16 @@
+# InterfaceUO_QP
+
+## IMPORTANT NOTE
+*** Any derived class of InterfaceUserObject does not support yet getMaterialProperty, please use auxvariables***
+
+## Description
+!syntax description /UserObjects/InterfaceUO_QP
+
+InterfaceUO_QP is a user object computing and storing average values across an interface for each quadrature. The kind of average value are the one available in [InterfaceAverageTools](/InterfaceAverageTools.md).
```

Comment:
for each quadrature -> __at__ each quadrature __point__
The kind of average value are the one available in -> The **various types** of **averages** are **described** in...